va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
@fhvwy ,  1. if there are Slice parameter, DPB is in the PPS, application  will parse slice haeader and set  active reference frame index is in slice parameter,  2. if there are no slice parameter from application, driver should parse the slice header and get the reference order (after ref_pic_list_modification or dec_ref_pic_marking